tests/test_22_04_2025.py

```diff
@@ -0,0 +1,984 @@
+    assert acl.getPermissionManager(EVM_SCRIPT_REGISTRY, REGISTRY_MANAGER_ROLE) == VOTING
+
+    REGISTRY_ADD_EXECUTOR_ROLE = web3.keccak(text="REGISTRY_ADD_EXECUTOR_ROLE")
+    assert acl.hasPermission(VOTING, EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE)
+    assert not acl.hasPermission(AGENT, EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE)
+    assert acl.getPermissionManager(EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE) == VOTING
+
+    # CuratedModule Permissions Transition
+    STAKING_ROUTER_ROLE = web3.keccak(text="STAKING_ROUTER_ROLE")
+    assert not acl.hasPermission(VOTING, CURATED_MODULE, STAKING_ROUTER_ROLE)
+    assert not acl.hasPermission(AGENT, CURATED_MODULE, STAKING_ROUTER_ROLE)
+    assert acl.getPermissionManager(CURATED_MODULE, STAKING_ROUTER_ROLE) == VOTING
+
+    MANAGE_NODE_OPERATOR_ROLE = web3.keccak(text="MANAGE_NODE_OPERATOR_ROLE")
+    assert not acl.hasPermission(VOTING, CURATED_MODULE, MANAGE_NODE_OPERATOR_ROLE)
+    assert acl.hasPermission(AGENT, CURATED_MODULE, MANAGE_NODE_OPERATOR_ROLE)
+    assert acl.getPermissionManager(CURATED_MODULE, MANAGE_NODE_OPERATOR_ROLE) == VOTING
+
+    SET_NODE_OPERATOR_LIMIT_ROLE = web3.keccak(text="SET_NODE_OPERATOR_LIMIT_ROLE")
+    assert acl.hasPermission(VOTING, CURATED_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE)
+    assert not acl.hasPermission(AGENT, CURATED_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE)
+    assert acl.getPermissionManager(CURATED_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE) == VOTING
+
+    MANAGE_SIGNING_KEYS = web3.keccak(text="MANAGE_SIGNING_KEYS")
+    assert acl.hasPermission(VOTING, CURATED_MODULE, MANAGE_SIGNING_KEYS)
+    assert not acl.hasPermission(AGENT, CURATED_MODULE, MANAGE_SIGNING_KEYS)
+    assert acl.getPermissionManager(CURATED_MODULE, MANAGE_SIGNING_KEYS) == VOTING
+
+    # Simple DVT Module Permissions Transition
+    assert acl.hasPermission(VOTING, SDVT_MODULE, STAKING_ROUTER_ROLE)
+    assert acl.hasPermission(AGENT, SDVT_MODULE, STAKING_ROUTER_ROLE)
+    assert acl.getPermissionManager(SDVT_MODULE, STAKING_ROUTER_ROLE) == VOTING
+
+    assert acl.hasPermission(VOTING, SDVT_MODULE, MANAGE_NODE_OPERATOR_ROLE)
+    assert not acl.hasPermission(AGENT, SDVT_MODULE, MANAGE_NODE_OPERATOR_ROLE)
+    assert acl.getPermissionManager(SDVT_MODULE, MANAGE_NODE_OPERATOR_ROLE) == VOTING
+
+    assert acl.hasPermission(VOTING, SDVT_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE)
+    assert not acl.hasPermission(AGENT, SDVT_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE)
+    assert acl.getPermissionManager(SDVT_MODULE, SET_NODE_OPERATOR_LIMIT_ROLE) == VOTING
+
+    # ACL Permissions Transition
+    CREATE_PERMISSIONS_ROLE = web3.keccak(text="CREATE_PERMISSIONS_ROLE")
+    assert acl.hasPermission(VOTING, ACL, CREATE_PERMISSIONS_ROLE)
+    assert not acl.hasPermission(AGENT, ACL, CREATE_PERMISSIONS_ROLE)
+    assert acl.getPermissionManager(ACL, CREATE_PERMISSIONS_ROLE) == VOTING
+
+    # WithdrawalQueue Roles Transition
```

Comment:
The order of the checks doesn't matches the [permissions transition plan](https://github.com/lidofinance/dual-governance/blob/a22a141ba331fa2407f13711126cb9d3691c4f83/docs/permissions-transition/permissions-transition-hoodi.md)